torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -27,6 +29,34 @@ def __setstate__(self, state):
         self.param = torch.ops.quantized.linear_prepack(*state[0])
         self.training = state[1]
 
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(PackedParameter, self)._save_to_state_dict(destination, prefix,
+                                                         keep_vars)
+        (w, b) = self.unpack()
+
+        destination[prefix + 'weight'] = w
+        destination[prefix + 'bias'] = b
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        weight = state_dict[prefix + 'weight']
+        bias = state_dict[prefix + 'bias']
+        self.param = torch.ops.quantized.linear_prepack(weight)
+        state_dict.pop(prefix + 'weight')
+        state_dict.pop(prefix + 'bias')
+
+        super(PackedParameter, self)._load_from_state_dict(state_dict, prefix,
+                                                           local_metadata,
+                                                           False, missing_keys,
+                                                           unexpected_keys,
+                                                           error_msgs)
+
+    def __repr__(self):
+        return repr(self.unpack())
+
+    def unpack(self):
+        return torch.ops.quantized.linear_unpack(self.param)
```

Comment:
Please add a test for loading and saving state dicts to verify the functionality here

Reply:
I created a task for this: #32189